HW_1/task_4.py

```diff
@@ -0,0 +1,10 @@
+# Задача 8: 
+
+n = int(input("Введите длину стороны n: "))
+m = int(input("Введите длину стороны m: "))
+k = int(input("Введите количество долек для отлома больше 0 : "))
+
+if k < n*m and ((k % n == 0) or (k % m == 0)): # определить кратность одной или другой стороны долькам и что долек не больше общего размера шоколадки
```

Comment:
```suggestion
if k < n * m and (k % n == 0 or k % m == 0): # определить кратность одной или другой стороны долькам и что долек не больше общего размера шоколадки
```
Также можно расширить первое условие по нижней границе допустимых значений переменной `k`